src/Login.js

```diff
@@ -16,8 +16,14 @@ const Login = () => {
     },
     style: {
       fontFamily: '"Helvetica New", Helvetica, sans-serif',
-      button: { backgroundColor: "#106ee9" },
-      input: { textColor: "#090909" },
+      borderRadius: 500,
```

Comment:
I think this should be `borderRadius: 10` or something else much smaller.

Reply:
Done